src/shims/unix/foreign_items.rs

```diff
@@ -58,6 +58,16 @@ pub trait EvalContextExt<'tcx>: crate::MiriInterpCxExt<'tcx> {
             // 512 seems to be a reasonable default. The value is not critical, in
             // the sense that getpwuid_r takes and checks the buffer length.
             ("_SC_GETPW_R_SIZE_MAX", |this| Scalar::from_int(512, this.pointer_size())),
+            // the machine's fd table does not start with a pre-allocated size but just the
+            // standard streams so we can't use it here and the spec imposes
+            // a minimum of `_POSIX_OPEN_MAX` (20).
+            ("_SC_OPEN_MAX", |this| Scalar::from_int(20, this.pointer_size())),
+            // The spec imposes a minimum of `_POSIX_ARG_MAX` (4096).
+            ("_SC_ARG_MAX", |this| Scalar::from_int(4096, this.pointer_size())),
+            // The spec imposes a minimum of `_POSIX_CHILD_MAX` (25).
+            ("_SC_CHILD_MAX", |this| Scalar::from_int(25, this.pointer_size())),
+            // 16 here is the least common denominator (i.e. macos, solarish).
+            ("_SC_NGROUPS_MAX", |this| Scalar::from_int(16, this.pointer_size())),
```

Comment:
Not sure how this is going to be helpful for any program Miri supports?

Reply:
that was just "anticipation" I planned try to use the value for [this line instead of the hardcoded 1](https://github.com/rust-lang/rust/blob/386a7c7ae22a3727c6a06633212d5a95edabdd74/library/std/src/os/unix/net/ancillary.rs#L99) to see if this help ; not urgent tough.

Reply:
`net` is anyway not supported by Miri, so seems a bit premature to add that constant already.